vanetza/asn1/its/Elevation.c

```diff
@@ -38,7 +38,7 @@ Elevation_constraint(const asn_TYPE_descriptor_t *td, const void *sptr,
  */
 #if !defined(ASN_DISABLE_OER_SUPPORT)
 static asn_oer_constraints_t asn_OER_type_Elevation_constr_1 CC_NOTUSED = {
-	{ 4, 0 }	/* (-4096..61439) */,
+	{ 2, 1 }	/* (-4096..61439) */,
```

Comment:
I see that the number of values fits in an `uint16_t`. Does asn1c already shift the values as expected, i.e. a raw 0 is decoded as -4096 in `Elevation_t`?

Reply:
Section 2.2 of https://www.oss.com/asn1/resources/books-whitepapers-pubs/Overview_of_OER.pdf summarizes the OER encoding rules for integer types. According to this summary, the Elevation type with a lower bound of -4096 and an upper bound of 61439 is encoded as a signed integer in four octets.